compiler/luci-interpreter/src/kernels/L2Normalize.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LUCI_INTERPRETER_KERNELS_L2NORMALIZE_H
+#define LUCI_INTERPRETER_KERNELS_L2NORMALIZE_H
+
+#include "core/Kernel.h"
+#include "core/KernelParams.h"
+
+namespace luci_interpreter
+{
+namespace kernels
+{
+
+class L2Normalize : public KernelWithParams<L2NormParams>
+{
+public:
+  L2Normalize(const Tensor *input, Tensor *output, const L2NormParams &params);
+
```

Comment:
@struss 
Sorry, I merged changes that caused conflicts here =_=
See https://github.com/Samsung/ONE/pull/2335/files#diff-fb2aa6e7e47ff3dba307e302d6f5a265R33 for details.

You need to rebase on to master and apply this change:
```suggestion

  std::vector<const Tensor *> getInputTensors() const override { return {_input}; }
  std::vector<Tensor *> getOutputTensors() const override { return {_output}; }
```

Reply:
Okay! I will add this and update a commit. never mind!